orienteer-core/src/main/java/org/orienteer/core/component/visualizer/IVisualizer.java

```diff
@@ -22,4 +20,6 @@
 	public boolean isExtended();
 	public Collection<OType> getSupportedTypes();
 	public <V> Component createComponent(String id, DisplayMode mode, IModel<ODocument> documentModel, IModel<OProperty> propertyModel, IModel<V> valueModel);
+	<V> Component createComponentForFiltering(String id, IModel<OProperty> propertyModel, IModel<V> valueModel);
```

Comment:
Non public? Please make it public and simplify name, for example to "createFilterComponent"